Fixes error
tst-pathopt.sh: /crosstool-0.20/build/powerpc-750-linux-gnu/gcc-2.95.3-glibc-2.2.2/build-glibc/elf/ld.so: cannot execute binary file
make[2]: *** [/crosstool-0.20/build/powerpc-750-linux-gnu/gcc-2.95.3-glibc-2.2.2/build-glibc/elf/tst-pathopt.out] Error 126
See also http://sources.redhat.com/ml/libc-alpha/2001-03/msg00113.html

===================================================================
RCS file: /cvs/glibc/libc/elf/Makefile,v
retrieving revision 1.196
retrieving revision 1.197
diff -u -r1.196 -r1.197
--- libc/elf/Makefile	2001/03/16 07:29:45	1.196
+++ libc/elf/Makefile	2001/03/25 04:57:51	1.197
@@ -246,8 +246,10 @@
 generated += $(addsuffix .so,$(strip $(modules-names)))
 
 ifeq (yes,$(build-shared))
+ifeq ($(cross-compiling),no)
 tests: $(objpfx)tst-pathopt.out
 endif
+endif
 
 $(objpfx)testobj1.so: $(libdl)
 $(objpfx)testobj1_1.so: $(objpfx)testobj1.so $(libdl)
